pkg/gardenlet/controller/shoot/shoot_control.go

```diff
@@ -505,12 +502,13 @@ func (c *Controller) durationUntilNextShootSync(shoot *gardencorev1beta1.Shoot)
 }
 
 func (c *Controller) updateShootStatusAndRequeueOnSyncError(g gardencore.Interface, shoot *gardencorev1beta1.Shoot, err error) (reconcile.Result, error) {
-	message := fmt.Sprintf("Shoot cannot be synced with Seed: %v", err)
-	c.recorder.Event(shoot, corev1.EventTypeNormal, gardencorev1beta1.EventOperationPending, message)
+	msg := fmt.Sprintf("Shoot cannot be synced with Seed: %v", err)
+	shootLogger := logger.NewShootLogger(logger.Logger, shoot.ObjectMeta.Name, shoot.ObjectMeta.Namespace)
+	shootLogger.Infof("[SYNC SHOOT] %s: %v", msg, err)
```

Comment:
`msg` already includes `err` and I think, this deserves `Error` log level ;)
```suggestion
	shootLogger.Error(err)
```